django/db/backends/base/operations.py

```diff
@@ -517,12 +518,20 @@ def convert_durationfield_value(self, value, expression, context):
         return value
 
     def check_aggregate_support(self, aggregate_func):
-        """Check that the backend supports the provided aggregate
+        warnings.warn(
+            "check_aggregate_support has been deprecated. Use "
+            "check_expression_support instead.",
+            RemovedInDjango20Warning, stacklevel=2)
```

Comment:
I think we might be too late to add more deprecation's to 1.8 (also missing deprecation docs).